src/domain/hanke/hankeView/HankeView.tsx

```diff
@@ -378,7 +378,7 @@ const HankeView: React.FC<Props> = ({
                 <CompressedAreaIndex
                   key={area.id}
                   area={area}
-                  haittaIndex={tormaystarkasteluTulos?.liikennehaittaIndeksi.indeksi}
+                  haittaIndex={tormaystarkasteluTulos?.liikennehaittaIndeksi?.indeksi}
```

Comment:
This was unrelated but necessary - at least for me local testing did not work without it